scrapers/errata.py

```diff
@@ -68,7 +69,55 @@ def update_neo4j(self, advisories):
                 'type_': advisory['type'],
                 'update_date': advisory['update_date'],
                 'updated_at': advisory['updated_at']
-            })[0]
+            }
+
+            associated_builds = self.get_associated_builds(advisory['id'])
+            if not associated_builds:
+                adv = Advisory.create_or_update(adv_params)[0]
+
+            for associated_build in associated_builds:
+                build = ContainerKojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+                if not build:
+                    build = KojiBuild.nodes.get_or_none(id_=associated_build['id_'])
+
+                container_adv = False
+                if build:
+                    if build.__label__ == 'ContainerKojiBuild':
```

Comment:
This is great, we just need to make sure any future changes to the scrapers don't create KojiBuild stubs without using the proper label.